integration-tests/src/test/java/tachyon/master/rawtable/RawTableMasterIntegrationTest.java

```diff
@@ -49,10 +51,9 @@ public final void before() throws Exception {
         mLocalTachyonClusterResource.get().getMaster().getInternalMaster().getFileSystemMaster();
   }
 
-  @Ignore
   @Test
   public void createRawTableTest() throws Exception {
-    mRawTableMaster.createRawTable(new TachyonURI("/testTable"), 1, null);
+    mRawTableMaster.createRawTable(new TachyonURI("/testTable"), 1, ByteBuffer.allocate(0));
```

Comment:
do we also need to make similar change (`null`  -> `ByteBuffer.allocate(0)`) in this file?